Cargo.toml

```diff
@@ -5,7 +5,7 @@ homepage = "https://github.com/multiformats/rust-multihash"
 
 keywords = ["multihash", "ipfs"]
 
-version = "0.4.2"
+version = "0.4.3"
```

Comment:
Can you remove this? I like to make version bumps its own commit, so I can tag it.